views/partials/email_partials/email-header.hbs

```diff
@@ -0,0 +1,23 @@
+<tr class="logo">
+  <td height="100"
+    style="height: 100px; background: #F9F9FA url('{{ SERVER_URL }}/img/email_images/monitor-logo-bg-f9f9fa.png') no-repeat 50%; background-size: 240px 50px;">
+  </td>
+</tr>
+<tr class="header">
+  <td>
+    <table style="width: 100%; height: 331px; background-color: #321C64; color: white; text-align: left;"
+      role="presentation" border="0" cellpadding="0" cellspacing="0">
+      <tr>
+        <td>
+          <h1>{{ heading }}</h1>
+          <p>{{{ subheading }}}</p>
+        </td>
+        <td class="header-image" style="width: 50%; background-color: #321C64; vertical-align: bottom;">
+          <img src="{{ SERVER_URL }}/img/email_images/person-at-desk.png" width="476" height="331"
```

Comment:
Assuming you mean this is broken on test emails sent via local.  It should work for preview page.  If you send to the Mailinator account the images show up, I believe (tho I don't exactly know why – I think it's because Gmail replaces images and Mailinator does not.) I confirmed it's working via Heroku.